### DISCUSSION TITLE
Question about querying related data

### DISCUSSION THREAD

**Discussion by fdjones** [prisma/prisma#8516] [Q&A] [answered]
I've gone through the docs (which are great btw!) but I'm not sure if this situation is covered - I'm trying to move my raw SQL queries over to Prisma as much as possible, but I have a problem. I have a SQL query which basically says `SELECT something  WHERE something.level = somethingElse.level` - how do I do this using Prisma? 

The prisma query I have will return a list of `something`'s (each of which have their own `level`), each with their own list of `somethingElse`'s - each of those `somethingElse`'s will also have a `level`. I want to include `somethingElse`'s ONLY if they have the same level as the `something` they are related to.

I'm not sure if my question is clear so apologies if not, but basically my question is about how to query data when your filtering not on a specific value like userId (which is being passed in to the query) but is instead filtering on some related value.

If that doesn't make any sense people let me know and I'll do my best to reword and explain further. Thanks :)

**Comment by ryands17** [comment, marked as answer]
@fdjones 👋 

Unfortunately this isn't possible currently with Prisma, so you would need to implement this with a raw query. We have a feature request for this so it would be great if you could add a 👍 so that we know the priority for this.
https://github.com/prisma/prisma/issues/5048

**Reply by fdjones** [reply]
Thanks @ryands17 , I gave the issue a 👍  :)

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
